sdk/tables/azure-data-tables/azure/data/tables/aio/_models.py

```diff
@@ -52,10 +52,7 @@ async def _get_next_cb(self, continuation_token, **kwargs):
 
     async def _extract_data_cb(self, get_next_return):
         self._location_mode, self._response, self._headers = get_next_return
-        props_list = [
-            TableItem._from_generated(t, **self._headers)  # pylint: disable=protected-access
-            for t in self._response.value
-        ]
+        props_list = [TableItem._from_generated(t) for t in self._response.value]  # pylint: disable=protected-access
```

Comment:
What was `self._headers` used for?

Reply:
It was not in use.
https://github.com/Azure/azure-sdk-for-python/blob/d5ed65730d3a1694a23ba9c9763767b852809ba1/sdk/tables/azure-data-tables/azure/data/tables/_models.py#L498-L500